Gemfile

```diff
@@ -34,4 +33,9 @@ gem 'unicorn'
 group :development, :test do
   gem 'rspec-rails', '~> 3.0'
   gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+end
+
+group :test do
+  gem 'codeclimate-test-reporter', require: nil
```

Comment:
Do we need this if Github does this as hooks? Or is it something extra? I've never used this gem, so I'm curious.